.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
Nit: We generally prefer to not put this in the project's `.gitignore` and instead leave it to each user to define these in their global git ignore.

Reply:
I remove this line out of `.gitignore`. :)